isign/isign.py

```diff
@@ -26,7 +27,7 @@
 
 
 class NotSignable(Exception):
-    """ This is just so we don't expose isign.app.NotSignable """
+    """ This is just so we don't expose other sorts of exceptions """
```

Comment:
I'm going to keep it as is, b/c there's code in Pantry that expects isign.NotSignable, not isign.exceptions.NotSignable.